hbshelpers/getDefaultUniversalLimit.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Generates a default result limit for each vertical on the universal page. The
+ * default is taken from the vertical's `universalLimit`, specified in the related
+ * page's VTC. If there is no such `universalLimit`, the back-end default of 10 will
+ * be used.
+ * 
+ * @param {Object} pageConfigs - The configurations for each page.
+ * @returns {Object} The default search.universalLimit for the page.
+ */
+ module.exports = function getDefaultUniversalLimit(pageConfigs) {
+  const universalLimit = Object.entries(pageConfigs)
+    .filter(([key, _]) => key != 'index')
```

Comment:
I think you can just omit the second variable in the tuple instead of using an underscore

Reply:
This is just a personal preference thing, but I actually prefer the underscore. I think it quickly communicates to the reader that I don't care about the other value. Seeing `.filter([key])` would give me pause for a second or too since I know `Object.entries` returns a tuple.

Reply:
unrelated to the first comment but github won't let me start a new thread. Could we rename `key` to maybe `pageName`?